test/PointItem.test.js

```diff
@@ -186,8 +186,8 @@ describe('have the correct classname for', () => {
     pointItem.setParent(parent);
     pointItem.redraw();
     assert.equal(pointItem.editable.updateTime, true);
-    assert.equal(pointItem.editable.updateGroup, undefined);
-    assert.equal(pointItem.editable.remove, undefined);
+    assert.equal(pointItem.editable.updateGroup, true);
+    assert.equal(pointItem.editable.remove, true);
   });
 
   it('an editable: {updateTime} override item (with boolean option false)', () => {
```

Comment:
These test cases were modified to match changes made at https://github.com/visjs/vis-timeline/pull/700/files#r504386157